base/applications/fontview/fontview.c

```diff
@@ -486,35 +484,6 @@ MainWnd_OnInstall(HWND hwnd)
         return -1;
     }
 
-    /* Open the fonts key */
-    res = RegOpenKeyExW(HKEY_LOCAL_MACHINE,
-                        L"SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Fonts",
-                        0,
-                        KEY_ALL_ACCESS,
-                        &hKey);
-    if (res != ERROR_SUCCESS)
-    {
-        MessageBoxW(hwnd, L"Failed top open the fonts key!", L"Debug1", MB_OK);
-        return -1;
-    }
-
-    /* Register the font */
-    res = RegSetValueExW(hKey,
-                         szFullName,
-                         0,
-                         REG_SZ,
-                         (LPBYTE)pszFileName,
-                         (wcslen(pszFileName) + 1) * sizeof(WCHAR));
-    if (res != ERROR_SUCCESS)
-    {
-        MessageBoxW(hwnd, L"Failed to register the new font!", L"Debug2", MB_OK);
-        RegCloseKey(hKey);
-        return -1;
-    }
-
-    /* Close the fonts key */
-    RegCloseKey(hKey);
-
```

Comment:
Yes so that is missing a call to AddFontResourceW(). Doesn't it also do the file copy, or does the file copy have to be done manually before?

Reply:
From MSDN (AddFontResourceA):     This function installs the font only for the current session. When the system restarts, the font will not be present. To have the font installed even after restarting the system, the font must be listed in the registry.